app/src/main/java/com/lagradost/cloudstream3/utils/UIHelper.kt

```diff
@@ -123,6 +128,35 @@ object UIHelper {
         )
     }
 
+    fun clipboardHelper(label: UiText, text: CharSequence) {
+        val ctx = context ?: return
+        try {
+            ctx.let {
+                val clip = ClipData.newPlainText(label.asString(ctx), text)
+                val labelSuffix = txt(R.string.toast_copied).asString(ctx)
+                ctx.getSystemService<ClipboardManager>()?.setPrimaryClip(clip)
+
+                if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.S_V2) {
+                    showToast("$label $labelSuffix")
```

Comment:
Nope, this is wrong, you cant use string like this on a UIText

Reply:
![Screenshot_20240308_015219](https://github.com/recloudstream/cloudstream/assets/147925818/eb354519-19f4-4270-a785-324da3a89a6a)

Reply:
![image](https://github.com/recloudstream/cloudstream/assets/125901294/a4202e77-c524-4dab-a67e-755660ad92e7)